src/main/java/seedu/address/ui/CompanyCard.java

```diff
@@ -49,6 +51,7 @@ public CompanyCard(Company company, int displayedIndex) {
         name.setText(company.getName().fullName);
         address.setText(company.getAddress().value);
         pocs.setText(company.getPocs().toString());
+        transactions.setText("Total: " + String.valueOf(company.getTotalTransacted()));
```

Comment:
I like this addition, it would definitely make the application more user friendly 👍